test/e2e/play_kube_test.go

```diff
@@ -803,6 +846,21 @@ var _ = Describe("Podman play kube", func() {
 
 	})
 
+	It("podman play kube fail with custom selinux label", func() {
```

Comment:
You have to add this to get pass the CI:
```golang
if !selinux.GetEnabled() {
	Skip("SELinux not enabled")
}
```
And make sure you import `github.com/opencontainers/selinux/go-selinux` at the top.

The test doesn't work on systems without selinux support like ubuntu.

Reply:
sorry hadn't considered cases where selinux is not enabled, have updated the test